CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,54 @@
+
+Harassment includes:
+
+* Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, or religion.
+* Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+* Deliberate misgendering or use of ‘dead’ or rejected names.
+* Gratuitous or off-topic sexual images or behaviour  in spaces where they’re not appropriate.
+* Physical contact and simulated physical contact (_e.g._, textual descriptions like “*hug*” or “*backrub*”) without consent or after a request to stop.
+* Threats of violence.
+* Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm.
+* Deliberate intimidation.
+* Stalking or following.
+* Harassing photography or recording, including logging online activity for harassment purposes.
+* Sustained disruption of discussion.
+* Unwelcome sexual attention.
+* Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+* Continued one-on-one communication after requests to cease.
+* Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect vulnerable people from intentional abuse.
+* Publication of non-harassing private communication.
+
+Shesmu developers prioritizes marginalized people’s safety over privileged people’s comfort. OICR GSI management reserves the right not to act on complaints regarding:
+
+* ‘Reverse’ -isms, including ‘reverse racism,’ ‘reverse sexism,’ and ‘cisphobia’
+* Reasonable communication of boundaries, such as “leave me alone,” “go away,” or “I’m not discussing this with you.”
+* Communicating in a ‘tone’ you don’t find congenial
+* Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions
+
+# Reporting
+If you are being harassed by a member of Shesmu developers, notice that someone else is being harassed, or have any other concerns, please contact the OICR GSI management at [email address or other contact point]. If the person who is harassing you is on the team, they will recuse themselves from handling your incident. We will respond as promptly as we can.
+
+This code of conduct applies to Shesmu developers spaces, but if you are being harassed by a member of Shesmu developers outside our spaces, we still want to know about it. We will take all good-faith reports of harassment by Shesmu developers members, especially OICR Genome Informatics, seriously. This includes harassment outside our spaces and harassment that took place at any point in time. The abuse team reserves the right to exclude people from Shesmu developers based on their past behavior, including behavior outside Shesmu developers spaces and behavior towards people who are not in Shesmu developers.
```

Comment:
Might not want to single out Genome Informatics here. :)

Reply:
I mean to write GSI. So, this was in the original where it's supposed to be _the management team_. I'm not sure it really makes sense. Or should I just write OICR or delete that part?

Reply:
How about "We will take all good-faith reports of harassment by the Shesmu development team seriously."?

Reply:
Works for me.